src/plugins/dashboard/public/dashboard_app/top_nav/add_panel_action_menu_items.ts

```diff
@@ -30,56 +51,50 @@ const onAddPanelActionClick =
     } else action.execute(context);
   };
 
-export type GroupedAddPanelActions = EuiContextMenuPanelDescriptor & {
-  icon?: string;
-};
-
-export const getAddPanelActionMenuItems = (
+export const getAddPanelActionMenuItemsGroup = (
   api: PresentationContainer,
   actions: Array<Action<object>> | undefined,
   closePopover: () => void
-): [EuiContextMenuPanelItemDescriptor[], Record<string, GroupedAddPanelActions>] => {
-  const ungrouped: EuiContextMenuPanelItemDescriptor[] = [];
```

Comment:
How are we handling embeddables/panel types that don't have a group defined? Can we guarantee all future panel types will have a group defined? Should we making having a group defined a strict requirement moving forward?

Reply:
The design spec doesn't leave accommodation for embeddables without a group. If we attempt rendering them, I suspect these embeddable would seem like they belong to whatever the proceeding group is, for this reason embeddables without a group aren't included. This is also stated in the PR description. I suppose it'd make sense then to make it a strict requirement to include a group

Reply:
Why don't we fall back to a default group?

Reply:
I think `Other` works the best, though I do like `hodgepodge` a lot hahah.